samples: dect_phy: dect_shell: sett: band 4 support (#19091)
nrfconnect/sdk-nrf: samples/dect/dect_phy/dect_shell/README.rst

     dect sett --tx_pwr -16

* Change the default band to ``2`` (has impact when automatic channel selection is used, in other words, when the set channel is zero in ``dect rssi_scan`` or in ``dect mac beacon_start`` command):

.. caution::

>    There might be region specific limitations for radio channel usage. See Regulations and Channel frequency sections from :ref:`nrf_modem_dect_phy` for using different DECT NR+ radio bands and channels in different regions.
>    Channel should be always measured with ``dect rssi_scan`` -command before accessing the band.

Reviewer: ```suggestion
   There might be region-specific limitations for radio channel usage.
   See Regulations and Channel frequency sections of the :ref:`nrfxlib:nrf_modem_dect_phy` page for using different DECT NR+ radio bands and channels in different regions.
   Make sure to always measure the channel with the ``dect rssi_scan`` command before accessing the band.
```

Reviewer: You might want to move this caution after the example command.
